conda-reqs/firesim.yaml

```diff
@@ -101,7 +101,6 @@ dependencies:
     # conda environment, pip's dependency resolution can conflict with
```

Comment:
commenting here because idk where else to put this, but we also need to change the conda-lock-modified check in CI to check these new files since they've been moved / split up.

Reply:
Done